styles/colors-base.less

```diff
@@ -90,6 +90,10 @@
 @agate-button-highlighted-bg-image: inherit;
 @agate-button-selected-bg-color: inherit;
 @agate-button-selected-color: inherit;
+@agate-button-lightOpaque-color: inherit;
```

Comment:
going to consolidate all the sub-themes into the base since there's no variation.